src/sections/Navigation/index.js

```diff
@@ -11,6 +12,102 @@ import ScrollspyMenu from "./utility/ScrollspyMenu.js";
 import NavigationWrap from "./navigation.style";
 
 const Navigation = () => {
+    const data = useStaticQuery(
+        graphql`
+            query  {
+                Projects: allMdx(
+                    sort: { fields: [frontmatter___date], order: DESC }
+                    filter: { fields: { collection: { eq: "projects" } } }
+                    limit: 2
+                ) {
+                    nodes {
+                        id
+                        frontmatter {
+                            title
+                            thumbnail{
+                                publicURL
```

Comment:
Here childImageSharp should be used like (reference can be taken from other queries)